src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -11,9 +11,42 @@
  * Реализуйте метод moveRobot()</p>
  */
 public class FieldXY {
+    private static int MIDDLE_OF_BOARD = 0;
```

Comment:
you don't need constant for zero

Reply:
but in checklist was point Don’t use magic numbers in your project. Let’s replace them with constants. Remember about correct naming. 0 is magic number.